pallets/common/src/lib.rs

```diff
@@ -249,21 +325,34 @@ pub mod pallet {
 		+ TypeInfo
 		+ account::Config
 	{
+		/// Weight info.
 		type WeightInfo: WeightInfo;
+
+		/// Events compatible with [frame_system::Config::Event].
 		type Event: IsType<<Self as frame_system::Config>::Event> + From<Event<Self>>;
 
+		/// Currency.
```

Comment:
Probably should clarify everything here and its uses, beyond just name repeating :)

Like, WeightInfo could have `Weight information for functions of this pallet.`, CollectionDispatcher could have `Dispatcher of operations on collections.`, ContractAddress `Address under which the CollectionHelper contract would be available.` (I think?), etc.

Reply:
Well, if you insist.

Currency: `Handler of accounts and payment.`
CollectionCreationPrice: `Set price to create a collection.`
TreasuryAccountId: `Account which holds the chain's treasury.`

And also some small clarifications on these would be nice:
EvmTokenAddressMapping: `Mapper for token addresses to Ethereum addresses.`
CrossTokenAddressMapping: `Mapper for token addresses to [`CrossAccountId`].`